tests/test_error_cases.py

```diff
@@ -124,7 +124,7 @@ def compatible_catalogs(self):
             cpus=cpus,
             add_noise=add_noise,
         )
-        draw_output = next(draw_generator)  # noqa: F841
+        _ = next(draw_generator)  # noqa: F841
```

Comment:
Is there a reason why you don't just call `next(draw_generator)` ?

Reply:
good catch!